### ISSUE TITLE
local db or local Q & A

### ISSUE THREAD

**Issue by miguelfs** [yeasir01/trivia-cli#1]
Hi, great user interface. Exactly what I was looking for.

Would it be possible to plug a local db or local set of questions? Instead of looking remotely at open trivia dataset.

**Comment by yeasir01** [comment]
Yes, it's possible, but it might not be worth the effort since a large part of the codebase would need significant changes. If you're just building something simple, you could structure it as an array of objects like this:

```
var questions = [
    {
        title: "What temperature does water boil at?",
        choices: ["100c", "150C", "200c", "212c"],
        answer: "100c"
    },
    {
        title: "What is often seen as the smallest unit of memory?",
        choices: ["Kilobyte", "Megabyte", "Byte", "Gigabyte"],
        answer: "Byte"
    }
];
```

you can check out one of my older projects for an idea on how to structure the code.
https://github.com/yeasir01/timed-quiz
